docs/README.md

```diff
@@ -0,0 +1,91 @@
+# DSP-APP Documentation
+
+This is the DSP-APP documentation, based on [MkDocs](https://www.mkdocs.org) and published
+under [http://dasch-swiss.github.io/dsp-app](http://dasch-swiss.github.io/dsp-app).
+
+## Contribute
+
+If you would like to add your own contributions to the docs, please read the following information regarding the file structure to ensure you follow the same structure.
+
+### File structure
+
+The documentation consists of three main topics with subordinate themes:
+
+1. **index** contains all information about the DSP-APP
+1. **how-to-use** contains all about the usage of the DSP-APP modules
```

Comment:
"contains all about" > weird to me
do you mean "all the information about..."

Reply:
yes it's weird... I'll fix it

Reply:
but as you said, "the big refactoring will be part of another PR." So you could aadd this point to the other PR.

Reply:
> but as you said, "the big refactoring will be part of another PR." So you could aadd this point to the other PR.

sure... but the content came from DSP-UI documentation and has to be updated for DSP-APP. DSP-APP documentation has another structure than DSP-UI

resolved it in 4440fc4